Lib/fontParts/base/kerning.py

```diff
@@ -116,28 +185,37 @@ def _round(self, multiple=1):
     # -------------
 
     def interpolate(
-        self, factor, minKerning, maxKerning, round=True, suppressError=True
-    ):
-        """
-        Interpolates all pairs between two :class:`BaseKerning` objects:
+        self,
+        factor: TransformationType,
+        minKerning: MathKerning,
```

Comment:
Ah, there is the same thing here of MathKerning really coming from the private method, and maybe that import should stay in the private method so it can get overwritten, but then I do not know what to do for type here